scanner/updater/export.go

```diff
@@ -22,18 +22,23 @@ import (
 // Export is responsible for triggering the updaters to download Common Vulnerabilities and Exposures (CVEs) data
 // and then outputting the result as a zstd-compressed file with .ztd extension
 func Export(ctx context.Context, outputDir string) error {
-
 	err := os.MkdirAll(outputDir, 0700)
 	if err != nil {
 		return err
 	}
 	// create output json file
-	outputFile, err := os.Create(filepath.Join(outputDir, "output.json.ztd"))
+	outputFile, err := os.Create(filepath.Join(outputDir, "output.json.zst"))
 	if err != nil {
 		return err
 	}
+	defer func() {
+		closeErr := outputFile.Close()
+		if closeErr != nil {
+			zlog.Error(ctx).Err(closeErr).Msg("Failed to close output file")
+		}
```

Comment:
```suggestion
		if err := outputFile.Close(); err != nil {
			zlog.Error(ctx).Err(err).Msg("Failed to close output file")
		}
```

This won't overwrite the original `err`, as the new `err` is only bound to the `if`

Reply:
Link to proof: https://go.dev/play/p/Q-7OLANJPVv